t/03_memleak.t

```diff
@@ -5,6 +5,14 @@ use Linux::Socket::Accept4;
 use IO::Socket::INET;
 use IO::Select;
 
+my $statm_path = '/proc/self/statm';
+
+if ($^O eq 'freebsd')
+{
+    substr($statm_path, 0, 0, '/compat/linux');
+    -e $statm_path or die "linproc has to be mounted in /compat/linux/proc\n";
```

Comment:
Test should not die if `linproc` is not mounted. Because `die` in test means test failure.
I think we should use `skip` in this case.